aptos-move/move-examples/fungible_asset/fa_coin/sources/FACoin.move

```diff
@@ -1,7 +1,7 @@
 /// A 2-in-1 module that combines managed_fungible_asset and coin_example into one module that when deployed, the
 /// deployer will be creating a new managed fungible asset with the hardcoded supply config, name, symbol, and decimals.
 /// The address of the asset can be obtained via get_metadata(). As a simple version, it only deal with primary stores.
```

Comment:
` it only deal with primary stores.` => ` it only deals with primary stores.`

sorry for all the separate typo fixes, think this is the last one!